$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_gcc_texi,v 1.1 2009/12/06 15:52:19 tg Exp $
--- gcc/doc/gcc.texi.orig	Wed Jul 30 05:28:30 2008
+++ gcc/doc/gcc.texi	Sun Dec  1 04:12:14 2013
@@ -65,8 +65,8 @@ Texts being (a) (see below), and with th
 @ifnottex
 @dircategory Software development
 @direntry
-* gcc: (gcc).                  The GNU Compiler Collection.
-* g++: (gcc).                  The GNU C++ compiler.
+* gcc-VSS: (gcc-VSS).          The GNU Compiler Collection.
+* g++-VSS: (gcc-VSS).          The GNU C++ compiler.
 @end direntry
 This file documents the use of the GNU compilers.
 @sp 1
